### PR TITLE
MoH - Fix table exports #48

### PR DESCRIPTION
**Pull request by djwoun**
# Description
The current export and column button is implemented by simply calling GridToolbar. It's a function the mui library that provides  with the basic functionalities export and column. I tried to see if the function was customizable. I didn't have too much luck, so I have shifted to just creating our own export csv function/button. 

## Relevant issue(s)
MoH - Fix table exports
[#48](https://github.com/hack4impact-utk/Maintenance-Team/issues/48)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have assigned reviewers to this PR

### CODE REVIEW

File: src/views/donationView/index.tsx [hunk 1]
```diff
@@ -96,6 +107,76 @@ export default function DonationView({ donations }: DonationViewProps) {
...
+          >
+            {allColumns.map((column) => (
+              <MenuItem key={column.field} value={column.field}>
+                <Checkbox checked={visibleColumns.includes(column.field)} />
+                <ListItemText primary={column.headerName} />
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+        <Button variant="contained" color="primary" onClick={exportToCSV}>
```
Comment (andlrutt) [inline src/views/donationView/index.tsx:141]: Can we use the outlined variant here? It's more of a secondary action if that makes sense

Comment (andlrutt) [reply src/views/donationView/index.tsx:141]: Also, if possible, reduce the size of the button, it looks a bit too tall. OK if this is not feasible, let me know either way

**Review by andlrutt — CHANGES_REQUESTED** [review]
Overall it seems to work great!

Take a peek at the story again, a few misses there:
- This fix needs to be implemented in three places. Maybe there's opportunity for a reusable component here?
- If possible, prevent the rows from being clickable. It no longer only exports the selected rows, but you can still select them, which has no function
- The exported csv should not contain a column for the edit icon

File: src/views/donationView/index.tsx (hunk 1 above)
Comment (djwoun) [reply src/views/donationView/index.tsx]: I have made the changes for 
-  prevent the rows from being clickable
- csv does not contain a column for the edit icon
- outlined variant
- reduce button size

**Review by andlrutt — APPROVED** [review]
Nicely done! We may want to break this out into a reusable component in the future but this will do just fine for MVP. Nice work!

**Pull request merged**